flumine/backtest/simulated.py

```diff
@@ -194,7 +194,14 @@ def _process_sp(self, publish_time: int, runner: RunnerBook) -> None:
             self._bsp_reconciled = True
             _order_type = self.order.order_type
             if _order_type.ORDER_TYPE == OrderTypes.LIMIT:
-                size = self.size_remaining
+                if self.side=='BACK':
+                    size = self.size_remaining
+                else:
+                    remaining_risk = (_order_type.price-1.0) * self.size_remaining
+                    size = round(remaining_risk / (actual_sp-1.0), 2)
+                    # Not sure if this is strictly accurate,
```

Comment:
Agreed its required, can you change this comment to or similar

```python
# cancel remaining size
```

Would be good to confirm that betfair do cancel the remaining size, might be lapsed instead?

Reply:
I have an example where zero size was matched before the SP was taken. The "orderType" field changed from "LIMIT" to "MARKET_ON_CLOSE" . The bet id did not change.

I'm not sure what would have happened if the bet was partially matched beforehand. It would be hard to create an example.

Reply:
Yeah it would be very tricky, shouldn't matter anyway as long its cancelled